docs/installation.rst

```diff
@@ -49,10 +49,32 @@ Now you have the executable *eProsima Micro XRCE-DDS Agent* installed in your sy
 .. important::
     The *eProsima Micro XRCE-DDS Agent* executable comes with a rich CLI.
     Find out all the options offered by this CLI when running the *Agent* in the :ref:`agent_cli` section of the
-    :ref:`micro_xrce_dds_agent_label` page. 
+    :ref:`micro_xrce_dds_agent_label` page.
 
-.. note::
-    If you wish, you can also install the Micro XRCE-DDS Agent as a `Snap package. <https://snapcraft.io/micro-xrce-dds-agent>`_ To do so, simply execute ``sudo snap install micro-xrce-dds-agent`` in your Linux console. You can add the ``--edge`` flag.
+Installation from Snap package
+******************************
+
+    If you wish, you can also install the *eProsima Micro XRCE-DDS Agent* as a `Snap package. <https://snapcraft.io/micro-xrce-dds-agent>`_
+
+    To do so, simply execute ``sudo snap install micro-xrce-dds-agent`` in your Linux console.
+    This will download the Snap package corresponding to the ``stable`` version, that is, *master* branch on GitHub.
+
+    You can add the ``--edge`` flag to the installation command, just in case you want to download the Snap image corresponding to the *develop* branch on the upstream source code repository.
+
+.. _install_agent_docker:
+
+Using the provided Docker image
+*******************************
+
+    The *eProsima Micro XRCE-DDS Agent* can also be launched directly from its dedicated `Docker image. <https://hub.docker.com/r/eprosima/micro-xrce-dds-agent>`_
```

Comment:
Sentence missing a final full stop.

Reply:
The full stop is included in the html reference :)